mycore-orcid2/src/main/resources/xsl/orcid2/v3/work2mcr.xsl

```diff
@@ -1,87 +1,138 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <xsl:stylesheet version="3.0"
-  xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
-  xmlns:mods="http://www.loc.gov/mods/v3"
-  xmlns:work="http://www.orcid.org/ns/work"
-  xmlns:common="http://www.orcid.org/ns/common"
-  exclude-result-prefixes="xsl">
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mods="http://www.loc.gov/mods/v3"
+                xmlns:work="http://www.orcid.org/ns/work"
+                xmlns:common="http://www.orcid.org/ns/common"
+                xmlns:fn="http://www.w3.org/2005/xpath-functions"
+                exclude-result-prefixes="fn xsl">
 
-  <xsl:import href="resource:xsl/orcid2/v3/work2mcr_generic.xsl" />
+  <xsl:import href="resource:xsl/orcid2/v3/work2mcr_generic.xsl"/>
+
+  <!--
+    To map an orcid type to mods:genre provide a classification containing the mapping. For each category provide a
+    label[@xml:lang='x-mapping']. The text attribute of that label contains the actual mapping like
+    $MCR.ORCID2.Genre.Mapping.Classification.Mapping.Prefix:$ORCID-Work-Type e.g. orcid:other
+
+    To determine the correct mods:genre of the host provide a mapping in a label[@xml:lang='x-hosts'] of each category.
+    The x-hosts attribute should only contain valid category ids with respect to the $MCR.ORCID2.Genre.Mapping.Classification.
+  -->
+  <xsl:param name="MCR.ORCID2.Genre.Mapping.Classification"/>
+  <xsl:param name="MCR.ORCID2.Genre.Mapping.Classification.Mapping.Prefix"/>
+  <xsl:param name="MCR.ORCID2.Genre.Mapping.Default.Genre"/>
+  <xsl:param name="MCR.ORCID2.Genre.Mapping.Standalone.xHost" select="'standalone'"/>
```

Comment:
I would also document the property in the mycore.properties.

Reply:
We already have a classification mapping mechanism in place.